docs/docs/configuration/server.rst

```diff
@@ -229,6 +229,84 @@ timeout
     specify a timeout in seconds for blocking operations like the
     connection attempt.
 
+mail_language
+	specify the language of the mail format by its `ISO 639-1<https://en.wikipedia.org/wiki/ISO_639-1>`_ (two letter) 
+	code, default to `en`.
+
+mail_title_admin
+	specify the title format of the notification email sent to the admin, default 
+	to `{title}`, in which `{title}` is the title of the thread. Available tags:
+	`{title}`, `{replier}`
+
+mail_title_user
+	specify the title format of the notification email sent to the subscribed
+	commenter, default to `Re: New comment posted on {title}`, in which `{title}`
+	is the title of the thread. Available tags: `{title}`, `{receiver}`,
+	`{replier}`
+
+admin_format_urluser_moderate
```

Comment:
Can we perhaps use a real templating system for the e-mails, e.g. jinja2 ? That way, we don't end up with a plethora of configuration variables, one for each combination of tags that is available.

Reply:
Okay, I'll try looking into that. I also deem that plain text is not really flexible.